modules/storage-rocksdb/src/main/java/org/apache/ignite/internal/storage/rocksdb/PartitionDataHelper.java

```diff
@@ -263,6 +294,55 @@ static BinaryRow deserializeRow(ByteBuffer buffer) {
         return new BinaryRowImpl(schemaVersion, binaryTupleSlice);
     }
 
+    byte[] createPayloadKey(ByteBuffer dataId) {
+        byte[] result = PAYLOAD_KEY_BUFFER.get()
+                .clear()
+                .putInt(tableId)
+                .putShort((short) partitionId)
+                .put(dataId)
+                .array();
+
+        dataId.rewind();
+
+        // Always use 0 for the last bit (tombstone flag), because it only makes sense when data ID is stored as a value.
+        setLastBit(result, result.length - 1, false);
+
+        return result;
+    }
+
+    /**
+     * Changes the last bit of the byte identified by an index in an array.
```

Comment:
It's a first bit, not last bit. What exactly did you mean in this name?

Reply:
it's the last bit if you count from the opposite side) Let's rename it to `setFirstBit`

Reply:
We could use setRightMostBit, or setLsb / setLeastSignificantBit to avoid confusion

Maybe even "setTombstoneBit"